File: api/src/Quiniela.Api/Entities/Round.cs
using Quiniela.Api.Enums;

namespace Quiniela.Api.Entities;

public class Round
{
    public Guid Id { get; set; }
    public Guid BoardId { get; set; }
    public string Name { get; set; } = string.Empty;
    public RoundStatus Status { get; set; }
    public DateTime? StartDateTime { get; set; }
    public DateTime? EndDateTime { get; set; }
    public DateTime CreatedAt { get; set; }
}

File: src/app/features/rounds/models/round.ts
export enum RoundStatus {
  Draft = 0,
  Open = 1,
  Active = 2,
  Completed = 3,
}

export interface Round {
  id: string;
  boardId: string;
  name: string;
  status: RoundStatus;
  startDateTime: string;
  endDateTime: string;
  createdAt: string;
}
